qemu/tests/nested_test.py

```diff
@@ -62,13 +63,26 @@ def generate_parameter_file(params):
         guest_password = params.get("password")
 
         bootstrap_options = params.get("nested_bs_options")
+        accept_cancel = params.get("accept_cancel", False)
+
         kar_cmd = "python3 ./ConfigTest.py "
 
+        host_cpu_models = virttest_cpu.get_host_cpu_models()
+        logging.info("host_cpu_model: %s" % host_cpu_models)
+
         test_type = params.get("test_type")
-        if test_type:
+        variant_name = params.get("nested_test")
+        if variant_name == "boot_l2":
             case_name = params.get("case_name")
             if case_name:
                 kar_cmd += " --%s=%s " % (test_type, case_name)
+        prefix_case_name = "x86_cpu_model"
+        if variant_name == "check_cpu_model_l2":
+            case_name = params.get("case_name", "")
+            for i in host_cpu_models:
+                case_name += "%s.%s," % (prefix_case_name, i)
+            kar_cmd += " --%s=%s " % (test_type, case_name[:-1])
```

Comment:
Move `prefix_case_name` to the case `check_cpu_model_l2`.
```
        variant_name = params.get("nested_test")
        case_name = params.get("case_name", '')
        if variant_name == "check_cpu_model_l2":
            host_cpu_models = virttest_cpu.get_host_cpu_models()
            logging.info("host_cpu_model: %s" % host_cpu_models)
            for i in host_cpu_models:
                case_name += "%s.%s," % (prefix_case_name, i)
            case_name.strip(',')
        kar_cmd += " --%s=%s " % (test_type, case_name)
```

Reply:
Fixed. Thanks

Reply:
I do not think it be fixed.
1. `prefix_case_name` in cfg for case `check_cpu_model_l2`
2. No need to add the if block for 'boot_l2'
3. `kar_cmd` is out of all, the format is same for all cases.

Reply:
Fixed. Please review. Thanks